libs/model-assessment/src/lib/ModelAssessmentDashboard/Controls/ModelOverview/FeatureConfigurationFlyout.tsx

```diff
@@ -181,6 +188,24 @@ export class FeatureConfigurationFlyout extends React.Component<
 
   private onRenderFooterContent = () => {
     const tooManyFeaturesSelected = this._selection.getSelectedCount() > 2;
+    const noChangesInSelectedFeatures =

```

Comment:
nit: it might be better to have it without negation, eg:
selectedFeaturesChanged